### ISSUE TITLE
False positive CA1721 (PropertyNamesShouldNotMatchGetMethods)

### ISSUE THREAD

**Issue by mavasani** [dotnet/roslyn-analyzers#2085]
#### Analyzer package

Example: [Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers)

#### Package Version

Example: v2.6.3

#### Diagnostic ID

Example: [CA1721](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1721-property-names-should-not-match-get-methods)

#### Repro steps

```csharp
public class C
{
    public int Value { get; }
    public static int GetValue(int i) => i;
}
```

#### Actual behavior
`warning CA1721: The property name 'Value' is confusing given the existence of method 'GetValue'. Rename or remove one of these members.`

#### Expected behavior
No CA1721 as the `GetValue` method is static and the `Value` property is instance
